owl2query-owlapi/pom.xml

```diff
@@ -4,14 +4,14 @@
     <parent>
         <artifactId>owl2query</artifactId>
         <groupId>cz.cvut.kbss</groupId>
-        <version>0.6.1</version>
+        <version>0.6.2</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
     <artifactId>owl2query-owlapi</artifactId>
     <name>OWL2Query OWLAPI Adapter</name>
 
     <properties>
-        <net.sourceforge.owlapi.version>5.1.20</net.sourceforge.owlapi.version>
+        <net.sourceforge.owlapi.version>5.5.0</net.sourceforge.owlapi.version>
```

Comment:
Note that OWLAPI 5.5.0 does not support Java 8.

Reply:
Good point, reverted, will solve later.